src/generic/FormikControl.jsx

```diff
@@ -30,7 +30,7 @@ const FormikControl = ({
         onChange={handleChange}
         onBlur={handleBlur}
         onFocus={handleFocus}
-        isInvalid={fieldTouched && fieldError}
+        isInvalid={!!fieldTouched && !!fieldError}
```

Comment:
Out of curiosity, why was the forced boolean conversion (`!!`) necessary, here?

Reply:
fieldError is not boolean value

Reply:
This condition did not always work correctly, and fieldError is not a boolean value.